lib/everypolitician/popolo/event.rb

```diff
@@ -3,6 +3,20 @@ module Popolo
     class Events < Collection; end
     class Event < Entity
       attr_accessor :start_date, :end_date
+
+      def people
+        raise NotImplementedError unless classification == 'legislative period'
+        ids = popolo.memberships.select { |m| m.legislative_period_id == id }
+                    .map(&:person_id)
+        popolo.persons.select { |p| ids.include? p.id }
+      end
+
+      def organizations
+        raise NotImplementedError unless classification == 'legislative period'
+        ids = popolo.memberships.select { |m| m.legislative_period_id == id }
```

Comment:
The use of this code in several places implies there should be a `term.memberships` method that each of them uses.